### DISCUSSION TITLE
Sending Messages to Trilium via Apple Shortcuts

### DISCUSSION THREAD

**Discussion by soulsands** [TriliumNext/Trilium#3853] [Show and tell]
Hello everyone, I have created an Apple shortcut that can directly send messages to Trilium with the help of a custom request handler. It supports direct input, system input, and clipboard content, making it very convenient.
Please refer to [trilium-ios-shortcut](https://github.com/soulsands/trilium-ios-shortcut)  for more details.

In my opinion, this is a very useful tool that can complement Trilium's capabilities. Would you considered adding it to the README.md? @zadam

**Comment by zadam** [comment]
Hello, that seems useful, but seems more suitable to the [Show and Tell](https://github.com/zadam/trilium/discussions/categories/show-and-tell) as it involves some scripting.

**Comment by soulsands** [comment]
it supports voice input now

**Reply by dumblegore** [reply]
Hi, trying to install your solution. What is the syntax of my custom API-adress? Just the network address? Any additional Trilium parameters?

**Reply by soulsands** [reply]
> Hi, trying to install your solution. What is the syntax of my custom API-adress? Just the network address? Any additional Trilium parameters?

Hi, any string will be fine, the path to fill in the shortcut is `{your server}/custom/{the value of #customRequestHandler}`.
